crates/re_types/definitions/rerun/datatypes.fbs

```diff
@@ -1,18 +1,19 @@
+include "./datatypes/rgba32.fbs"; // Must come before material.fbs
```

Comment:
why? `material.fbs` should just include `rgba32.fbs` (and then be listed here with the rest in alphabetical order). I thought the includes here have `pragma once` behavior.

Reply:
TIL